packages/hash/frontend/src/blocks/page/create-format-plugins/marks-tooltip.tsx

```diff
@@ -34,88 +73,96 @@ export const MarksTooltip: FunctionComponent<MarksTooltipProps> = ({
   focusEditorView,
   openLinkModal,
 }) => {
-  const getMarkBtnStyle = (name: string): CSSProperties => {
-    const isActive = activeMarks.find((mark) => mark.name === name);
-
-    if (isActive) {
-      if (name === "link") {
-        return {
-          color: "#3B82F6",
-        };
-      }
-
-      return {
-        backgroundColor: "#3B82F6",
-        color: "#ffffff",
-      };
-    } else {
-      return {
-        backgroundColor: "#ffffff",
-        color: "#000000",
-      };
-    }
-  };
-
-  const handleToggleMark = (name: string) => {
-    if (name === "link") {
-      openLinkModal();
-    } else {
-      toggleMark(name);
-    }
-
-    focusEditorView();
-  };
+  const theme = useTheme();
 
   return (
-    <div
-      style={{
-        // @todo use shadows from MUI theme https://app.asana.com/0/1203179076056209/1203480105875518/f
-        boxShadow:
-          "0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",
-        left: "50%",
-        marginTop: "-0.25rem",
+    <Paper
```

Comment:
In the designs there's some spacing between the selected text and the tooltip.

Designs:
<img width="445" alt="image" src="https://user-images.githubusercontent.com/42802102/210553657-24188bbd-8a64-4e70-8016-937efb6dc1cc.png">

Implementation:
<img width="258" alt="image" src="https://user-images.githubusercontent.com/42802102/210553671-823d2969-5976-465c-ac81-b5c2396b085c.png">

Reply:
Thank you for catching this, increased the spacing in 507a5eef9281ebeea8058e5a1f3c088f3f9a787c  between the selected text and the tooltip to 5px.